code/modules/tgui_panel/telemetry.dm

```diff
@@ -161,6 +161,7 @@
 	. += "<br><b>Telemetry Status:</b>"
 	switch(telemetry_status)
 		if(TGUI_TELEMETRY_STAT_NOT_REQUESTED)
+			message_admins("[key_name(client)] has denied a Telemetry Request and may have a modified client.")
 			. += "<span class='bad'>Telemetry Request Not Sent. Call a coder.</span>"
```

Comment:
The admins can see this anyway, since it shows when you check the player panel. There is no point in messaging all admins when the admin checking telemetry can see it anyway.